etna/commands/forecast_command.py

```diff
@@ -129,14 +129,13 @@ def forecast(
         freq_init = freq
         parse_dates = ["timestamp"]
 
-    df_timeseries = pd.read_csv(target_path, parse_dates=parse_dates)
-
+    df_timeseries = pd.read_csv(target_path, parse_dates=parse_dates, dtype={"segment": str})
```

Comment:
We should probably add some test that checks this behavior. Smth like en example in the issue.